modules/storages/app/controllers/storages/admin/storages_controller.rb

```diff
@@ -162,6 +165,20 @@ def update
     end
   end
 
+  def change_health_email_notifications
+    return head :forbidden unless %w[1 0].include?(permitted_storage_params[:health_notifications_enabled])
+
+    if @storage.update(health_notifications_enabled: permitted_storage_params[:health_notifications_enabled])
+      update_via_turbo_stream(component: Storages::Admin::Sidebar::HealthNotificationsComponent.new(storage: @storage.reload))
```

Comment:
Why are we reloading `@storage` here? It seems to be a non necessary request to me. We are under condition about successful update. Therefore `@stprage.health_notifications_enabled` should be up to date already.

Reply:
yeah, we should be able to do without it 👍🏾